figures/tasks.py

```diff
@@ -131,6 +131,52 @@ def populate_daily_metrics_for_site(site_id, date_for, ed_next=False, force_upda
                         force_update=force_update)
 
 
+@shared_task
+def populate_daily_metrics_for_site_async_sites(site_id, date_for,
+                                                ed_next=False, force_update=False):
+    """Collect metrics for the given site and date,
```

Comment:
@OmarIthawi Would you please provide a summary in the docstring? Why this function? Is it for Tahoe only, can it run in standalone? What do you want to jog your memory if you have to revisit this function several months from now?

Thanks!

Reply:
Reviewing in more depth, it appears that this function duplicates `populate_daily_metrics_for_site`, https://github.com/appsembler/figures/blob/main/figures/tasks.py#L100

 with the only change to add in logging, right? If this is the case, then let's just add the logging to `populate_daily_metrics_for_site` universally. Then we shouldn't need this new function

Reply:
Just to note, this would duplicate the `SITE:START` and `SITE:END` logging output when called by `populate_daily_metrics_next` _until_ we update the logging output in `populate_daily_metrics_next`. But I think this is acceptable for now